app/action/fluxActions.jsx

```diff
@@ -5,7 +5,7 @@ import dispatcher from '../dispatcher';
  * This function fetches all News Sources
  * @return {[type]} [description]
```

Comment:
You should specify what data type you are expecting this function to return in the Jdoc